pkg/network/protocols/tls/java/jattach_test.go

```diff
@@ -14,13 +14,24 @@ import (
 	"testing"
 	"time"
 
+	"github.com/cihub/seelog"
 	"github.com/stretchr/testify/require"
 
 	javatestutil "github.com/DataDog/datadog-agent/pkg/network/protocols/tls/java/testutil"
 	"github.com/DataDog/datadog-agent/pkg/network/testutil"
 	"github.com/DataDog/datadog-agent/pkg/util/kernel"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+func TestMain(m *testing.M) {
```

Comment:
added for debug?

Reply:
Yes. But it is useful to keep here IMO, so we can easily enable debug logs for java tests.